src/codeu/chat/server/Server.java

```diff
@@ -172,6 +163,44 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    // Clean - A client wants to clean the log
+    this.commands.put(NetworkCode.CLEAN_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+        File log = new File(persistentPath.getPath());
```

Comment:
put this into a Util function like TransactionLogUtil.clearLogs() , don't put any logic here.